auth/README.md

```diff
@@ -35,6 +35,11 @@ You can then run a given `ClassName` via:
     mvn exec:java -Dexec.mainClass=com.google.cloud.auth.samples.AuthExample
         -Dexec.args="compute"
 
+### Authenticating using an api key
```

Comment:
Would you mention what it does? Reference the title of the JSON file example above.